index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
+    <title>Achraf's Portfolio</title>
+  </head>
+
+  <body>
+    <header class="flex-div">
+      <h1>Welcome</h1>
```

Comment:
- [ ] the logo of the website should be in `a` tag not `h1` 
change `<h1>Welcome</h1>` to `<a href="#">Welcome</a>`

Reply:
Ahh Got it, Thank you so much, Sir.

Reply:
Hi @aimemalaika, I am done with the requirement changes Sir.